threenums.c

```diff
@@ -0,0 +1,49 @@
+/* Drew French */
+
+/* Does a bunch of stuff
+ * with 3 numbers given
+ * by the user
+ */
+
+# include <stdio.h>
+int a;
+int b;
+int c;
+int d;
+int sum;
+
+int main()
+{	
+	printf("\nEnter a value for a: ");
+	scanf("%d", &a);
+	printf("Enter a value for b: ");
+	scanf("%d", &b);
+	printf("Enter a value for c: ");
+	scanf("%d", &c);
+	
+	if(a == b && b == c)
+	{
+		printf("\nnumbers are equal\n");
+	}
+	else
+	{
+		if(a > b && a > b)
+		{
+			d = a;
+		}
+		if(b > a && b > c)
+		{
+			d = b;
+		}
+		if(c > a && c > b)
+		{
+			d = c;
+		}
```

Comment:
Nice job, but because of the way your logic is implemented, your code doesn't properly handle cases where two of the numbers are equal and also the largest number. Make sure to test for edge cases.